src/Tizen.NUI.Components/Controls/ScrollableBase.cs

```diff
@@ -523,7 +547,7 @@ public ScrollableBase() : base()
                 Layout = new AbsoluteLayout() { SetPositionByLayout = false },
             };
             ContentContainer.Relayout += OnScrollingChildRelayout;
-            propertyNotification = ContentContainer.AddPropertyNotification("position", PropertyCondition.Step(1.0f));
+            propertyNotification = ContentContainer.AddPropertyNotification("position", PropertyCondition.Step(0.000001f));
```

Comment:
I think it would be better to make this "0.000001f" value as named constant value to enhance code-readability. ^^;

Reply:
DONE